pulsar-metadata/src/main/java/org/apache/pulsar/metadata/bookkeeper/PulsarLedgerIdGenerator.java

```diff
@@ -173,7 +174,7 @@ private CompletableFuture<Long> generateLongLedgerId() {
                                 if (log.isDebugEnabled()) {
                                     log.debug("DELETING HIGH ORDER DIR: {}", path);
                                 }
-                                store.delete(path, Optional.of(0L));
```

Comment:
this is a relative path we build ourself and won't have the problem of carrying root path when getting path from `stats.getPath()`?

Reply:
Yes, you are right. Thanks for reminder.